pkg/referrerstore/oras/oras.go

```diff
@@ -83,11 +83,15 @@ func (s *orasStoreFactory) Create(version string, storeConfig config.StorePlugin
 
 	// Set up the local cache where content will land when we pull
 	if conf.LocalCachePath == "" {
-		conf.LocalCachePath = defaultLocalCachePath
+		homeDir, err := os.UserHomeDir()
+		if err != nil {
+			return nil, err
+		}
+		conf.LocalCachePath = filepath.Join(homeDir, defaultLocalCachePath)
```

Comment:
Do you want to use https://github.com/deislabs/ratify/tree/main/pkg/homedir

Reply:
switched to use `homedir.Get()`